mods/action.js

```diff
@@ -181,6 +181,13 @@ function buildEditResponse(res) {
 }
 
 ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
+    if (/\.test\.local$/.test(req.params.domain)) {
```

Comment:
I've just merged #324 so we can get rid of these hacks. We can discuss alternatives to it later. *beta* does seem like a good idea, especially now with shorter MW deployment trains.